### DISCUSSION TITLE
ZigbeeBridgePro new Firmware 20240710 flash fail

### DISCUSSION THREAD

**Discussion by Joylancer** [arendst/Tasmota#22099] [Support request]
Habe es heute mal versucht, die neue Firmware zu flashen.
Steps:
Update Tasmota auf die aktuelle DEV
Intel-Hex und Sonoff_flasher .be files aktualisiert
aktuelle Firmware auf die Bridge geladen (Danke an @s-hadinger )
Hier mal das Log dann dazu:
19:58:20.338 FLH: Starting verification of HEX file
19:58:39.820 FLH: Verification of HEX file OK
19:58:56.703 FLH: cc2652_flasher rx=23 tx=19 rst=15 bsl=22
19:58:56.729 FLH: Flashing started (takes 5-8 minutes during which Tasmota is unresponsive)
20:06:42.176 BRY: Exception> 'value_error' - len must be a multiple of 4
20:06:42.177 stack traceback:
20:06:42.178 : in function flash
20:06:42.179 : in function main

Da scheint noch was nicht zu stimmen.
Rückflashen auf 20220219 klappt problemlos.

**Comment by sfromis** [comment]
Looks like line 11186 of `SonoffZBPro_coord_20240710.hex` is not for a multiple of 4 bytes.
> `:0DBAD8000CFF0400014503011E483033003F`
